### PR TITLE
update weekly management goal

### PR DESCRIPTION
**Pull request by Krisocer**

### CODE REVIEW

File: management/Jersey_Shores.md [hunk 1]
```diff
@@ -35,9 +35,9 @@
 2. 11/02 - 11/08
 	* Manager: Zhimin Li
 	* Goals:
-		1. ...
-		1. ...
-		1. ...
+		1. Work on screen border collisions
+		2. Work on stemmer and add stopwords item
```
Comment (nfarnan) [inline management/Jersey_Shores.md:39]: Specifically what word do you hope to get done this week on the stemmer and stopwords?

File: management/Jersey_Shores.md [hunk 2]
```diff
@@ -35,9 +35,9 @@
...
+		3. Work on UI and python model
```
Comment (nfarnan) [inline management/Jersey_Shores.md:40]: Specifically what UI work?
What Python model are you referring to?

**Review by nfarnan — CHANGES_REQUESTED** [review]

File: management/Jersey_Shores.md (hunk 1 above)
Comment (Krisocer) [reply management/Jersey_Shores.md:39]: Using crates(you have allowed our NLP group to use) to implement stemmer and stopwords.

File: management/Jersey_Shores.md (hunk 2 above)
Comment (Krisocer) [reply management/Jersey_Shores.md:40]: For the python model: We are not actually implementing the python model in our game, and we just use it to do some tests to finish algorithm research.

For the UI: I am working on getting rover to appear and disappear on the screen, as well as getting his dialogue and the user input to show up on the screen

**Review by nfarnan — APPROVED** [review]

**Pull request merged**